pkg/reconciler/v1alpha1/route/reconcile_resources.go

```diff
@@ -70,6 +67,22 @@ func (c *Reconciler) getClusterIngressForRoute(route *v1alpha1.Route) (*netv1alp
 	return ingresses[0], nil
 }
 
+func routeOwnerLabelSelector(route *v1alpha1.Route) labels.Selector {
+	return labels.Set(map[string]string{
+		serving.RouteLabelKey:          route.Name,
+		serving.RouteNamespaceLabelKey: route.Namespace,
+	}).AsSelector()
+}
+
+func (c *Reconciler) deleteClusterIngressForRoute(route *v1alpha1.Route) error {
```

Comment:
deleteClusterIngressesForRoute?  Or maybe deleteClusterIngresses, since ForRoute is implied by the param.

Reply:
Done.